scala/scala.bzl

```diff
@@ -16,6 +16,7 @@
 
 
 _scala_filetype = FileType([".scala"])
+_srcjar_filetype = FileType([".srcjar"])
```

Comment:
I think we want `_scala_filetype = FileType([".scala", ".srcjar"])` and put the srcjar and scala files in srcs (to follow the java convention).